Jellyfin.Api/Controllers/SubtitleController.cs

```diff
@@ -0,0 +1,344 @@
+﻿using System;
```

Comment:
```suggestion
#nullable enable
#pragma warning disable CA1801

﻿using System;
```

Reply:
I'm not getting a CA1801 warning. Is there any reason it should be disabled?

Reply:
I get the warning:
```
SubtitleController.cs(256, 29): [CA1801] Parameter index of method GetSubtitlePlaylist is never used. Remove the parameter or use it in the method body.
```

Reply:
You are absolutely right. Now I remember seeing that warning and wondering why it is in the route but never used in the method. But somehow now the warning is gone for me.
Should we maybe add a todo to remove the index parameter as it is not used and introduce a breaking Api change in the future?

Reply:
You can add a TODO, but keep in mind that we plan on redoing the API after the conversion is complete